qutebrowser/utils/utils.py

```diff
@@ -42,6 +42,10 @@
 from PyQt5.QtCore import QUrl, QVersionNumber
 from PyQt5.QtGui import QClipboard, QDesktopServices
 from PyQt5.QtWidgets import QApplication
+if sys.version_info >= (3,9):
```

Comment:
```suggestion
if sys.version_info >= (3, 9):
```

Also, maybe add a comment mentioning that we need the `.files()` API which was added in 3.9 only?